tests/lib/static_content.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python
```

Comment:
Is it necessary to have a separate file for this `CONFIGLET_CONTENT` variable? 
Couldn't we put that variable inside `lib/cvaas_configlet.py`, so we can delete that file? 

Then we could create 1 file per static data we need as mentioned in the README: 
* `lib/cvaas_configlet.py`
* `lib/cvaas_container.py`
* `lib/cvaas_device.py`

Reply:
It is for code readability:

If you want to use `CONFIGLET_CONTENT` in your configlet structure named `SYSTEM_CONFIGLETS_TESTS`, python requires to first declare `CONFIGLET_CONTENT` and then `SYSTEM_CONFIGLETS_TESTS`. But considering, `CONFIGLET_CONTENT` is 300 lines long, it is more complex to locate our test values.

So, it is the reason why `CONFIGLET_CONTENT` is in `static_content.py`: we just have to import in our test case.

Idea would be to have as you mentionned:

- `lib/cvaas_configlet.py` for configlet test data
- `lib/cvaas_container.py` for container test data
- `lib/cvaas_device.py` for device test data
- `+ lib/static_content` for static content to use in test data above